spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/ModuleSerializer.scala

```diff
@@ -211,3 +211,9 @@ private case class GenericTypeWrapper[T: ClassTag](tensor : Tensor[T],
   ttpe : T
   )(implicit ev: TensorNumeric[T])
 
+// ModelTag is an unique identifier for a serialized model
+case class ModuleTag(publisher : String,
```

Comment:
Do we need a ModuleTag class? Or maybe just use a string which can contain arbitrary tag, e.g., bigdl.v_0_3_0.ssd?

Reply:
I discussed with @SeaOfOcean ,  Maybe with a ModuleTag we could handle the compatibility in a more elegant way in application (model_zoo or SSD model)